sentry/src/main/java/io/sentry/cache/EnvelopeCache.java

```diff
@@ -416,6 +425,11 @@ public void discard(final @NotNull SentryEnvelope envelope) {
     return ret.iterator();
   }
 
+  @Override
+  public synchronized boolean containsFile(@NotNull File file) {
```

Comment:
`m`: how about we rename this to something like `hasAlreadyProcessedFile(file)` to make it a bit less confusing on the calling side (where we actually processed cached files, but at the same time early-returning if it's already cached 😅 )